app/src/componentLibrary/attributeList/editableAttribute/attribute/attribute.jsx

```diff
@@ -55,6 +71,8 @@ export const Attribute = ({
         tabIndex={-1}
         className={cx('label', variant, { disabled })}
         onKeyDown={disabled ? null : handleAttributeKeyValueKeyDown}
+        onFocus={onFocus}
```

Comment:
It works fine, but only when the key:value in focus.
What about cross icon?

Reply:
Done.